superheroes.yaml

```
@@ -197,6 +201,7 @@ scripts:
     - sh: >
         ${{CONTAINER_RUNTIME}} run -d
         --name rest-villains
+        --privileged
```

This is going to run all modes with `--privileged`.  I think we should instead create a `CONTAINER_RUNTIME_PARAMS` state variable, and set that for the specific modes,
 e.g. in `semeru.build.script.yaml` we can set:
`CONTAINER_RUNTIME_PARAMS: --privileged`
and this could be;
```suggestion
        ${{ CONTAINER_RUNTIME_PARAMS }}
```

For the other modes, we can set this to be an empty string